src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -5446,12 +5467,18 @@ public sealed partial class MethodImplAttribute : System.Attribute
         public System.Runtime.CompilerServices.MethodImplOptions Value { get { return default(System.Runtime.CompilerServices.MethodImplOptions); } }
     }
     [System.FlagsAttribute]
+    [System.Runtime.InteropServices.ComVisibleAttribute(true)]
```

Comment:
The ComVisible attributes are useless in reference assemblies. They should be omitted.